integration_tests/integration_tests/test_table_monitoring_query.py

```diff
@@ -0,0 +1,138 @@
+from datetime import datetime
+from dbt_osmosis.core.osmosis import DbtProject
+from parametrization import Parametrization
```

Comment:
What's the benefit of `pytest-parametrization` over Pytest's built-in functionality?

Reply:
It's actually a wrapper on top of pytest's functionality.
Makes it a bit easier to provide names for all the cases + the default parameters is nice.
Not a must but I find it a bit more readable than pytest's existing functionality